test/e2e/delete_test.go

```diff
@@ -51,6 +51,9 @@ func (tc *testContext) clearWindowsInstanceConfigMap() error {
 
 // testBYOHRemoval tests that nodes are properly removed and deconfigured after removing ConfigMap entries
 func (tc *testContext) testBYOHRemoval(t *testing.T) {
+	if gc.numberOfBYOHNodes == 0 {
+		t.Skip("BYOH testing disabled, nothing to remove")
```

Comment:
BYOH testing is not really disabled so I would rather say `No BYOH nodes present`

Reply:
Makes sense.